plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -80,12 +80,20 @@ const getVisibleTabs = ( selectedTab: string ) => {
 
 	return currentVisibleTabs;
 };
-
 const renderTabs = (
 	marketplaceContextValue: MarketplaceContextType,
 	visibleTabs: Tabs
 ) => {
 	const { selectedTab, setSelectedTab } = marketplaceContextValue;
+
+	const onTacClick = ( tabKey: string ) => {
+		if ( tabKey === selectedTab ) {
+			return;
+		}
+		setSelectedTab( tabKey );
+		setUrlTabParam( tabKey );
+	};
```

Comment:
Nice! `onTacClick` looks like a typo – do we mean `onTabClick`?

Reply:
Thanks! I updated it.